internal/distributed/datacoord/client/client.go

```diff
@@ -629,3 +629,17 @@ func (c *Client) MarkSegmentsDropped(ctx context.Context, req *datapb.MarkSegmen
 	}
 	return ret.(*commonpb.Status), err
 }
+
+// BroadCastAlteredCollection is the DataCoord client side code for BroadCastAlteredCollection call.
+func (c *Client) BroadCastAlteredCollection(ctx context.Context, req *milvuspb.AlterCollectionRequest) (*commonpb.Status, error) {
+	ret, err := c.grpcClient.ReCall(ctx, func(client interface{}) (interface{}, error) {
```

Comment:
1) need some log here when successfully BroadCast
2) should be Broadcast , rethink the name also, if only 1 datacoord, why name it broadcast, notifyColletionChange?